reana_job_controller/job_monitor.py

```diff
@@ -194,6 +195,15 @@ def get_job_status(self, job_pod):
                             )
                         )
                         status = "failed"
+                        break
+                else:
+                    logging.warning(
+                        "Kubernetes job {} failed due to unkown issues".format(
```

Comment:
(1) Note `unknown` typo.

(2) Saw this exact warning happening after workflow finishes successfully/unsuccessfully:

```console
$ kubectl logs reana-run-batch-fb036d7e-9f22-4843-b63b-8f8e85e3ec1f-tg6vc workflow-engine
2020-10-16 11:38:59,826 | root | MainThread | INFO | Publishing step:0, cmd: python "code/helloworld.py" --inputfile "data/names.txt" --outputfile "results/greetings.txt" --sleeptime 0, total steps 1 to MQ
2020-10-16 11:39:05,856 | root | MainThread | INFO | Workflow fb036d7e-9f22-4843-b63b-8f8e85e3ec1f finished. Files available at /var/reana/users/00000000-0000-0000-0000-000000000000/workflows/fb036d7e-9f22-4843-b63b-8f8e85e3ec1f.
``` 

but:

```console
$ kubectl logs reana-run-batch-fb036d7e-9f22-4843-b63b-8f8e85e3ec1f-tg6vc job-controller
...
2020-10-16 11:38:59,834 | root | kubernetes_job_monitor | INFO | New Pod event received: ADDED
2020-10-16 11:38:59,835 | root | kubernetes_job_monitor | INFO | New Pod event received: ADDED
2020-10-16 11:38:59,835 | root | kubernetes_job_monitor | WARNING | Kubernetes job reana-run-job-abcaaf2e-1860-4e50-b467-838a1de3993b failed due to unkown issues
2020-10-16 11:38:59,836 | root | kubernetes_job_monitor | INFO | Grabbing pod reana-run-job-abcaaf2e-1860-4e50-b467-838a1de3993b-zdhsk logs ...
2020-10-16 11:38:59,836 | root | kubernetes_job_monitor | INFO | Storing job logs: 1eee756b-ca6a-4483-b84f-56e213d0a34d
```